### DISCUSSION TITLE
Curious if I can use Oqtane

### DISCUSSION THREAD

**Discussion by remesq** [oqtane/oqtane.framework#4131] [Q&A]
Is it possible, with Oqtane, to create rich, complex forms with conditions for users and then capture the information on the back-end? Sending e-mails with the information and suggestions based on answers would be a bonus.

**Comment by sbwalker** [comment]
@remesq there are a number of different ways your question could be interpreted:

1. Does the Oqtane Framework contain a Form Builder? No
2. Could a Form Builder be developed for use in Oqtane? Yes
3. Has a developer already created a Form Builder product that can be used with Oqtane? Yes @leigh-pointer can provide more information
4. Are there any open source Form Builders available for Oqtane? The Survey module (https://github.com/oqtane/Oqtane.Survey) could be customized into a Form Builder (if you are a software developer)

**Reply by remesq** [reply]
Much appreciated! You are leading me in the right direction.

I looked at Dynamic Forms from @leigh-pointer, but it seemed like it was used on the back-end (at least the examples shown seemed to indicate that). But I will investigate further.

I'll also take a look at the Survey module.

Thank you.

**Reply by sbwalker** [reply]
You can also explore 2SXC as it offers a wide variety of capabilities @iJungleboy

**Reply by leigh-pointer** [reply]
@remesq i am happy to give a deeper dive into the module. For now, forms can be designed by administrators and users who have permission. Data can be captured from users with permission. Data can be presented in many ways and the presentation displayed by permission. Its is also possible to consume the data and use in another module or system.
